aya/src/programs/links.rs

```diff
@@ -28,6 +31,25 @@ pub trait Link: std::fmt::Debug + 'static {
     fn detach(self) -> Result<(), ProgramError>;
 }
 
+bitflags::bitflags! {
+    /// Flags passed to [`SockOps::attach()`].
```

Comment:
This doc comment is now inaccurate

Reply:
Ah, true.  I've switched it to the more general comment of "Program attachment flags."  Let me know if you'd like a more descriptive comment.